### PR TITLE
fix: paste styles shortcut

### PR DESCRIPTION
**Pull request by dwelle**
fix https://github.com/excalidraw/excalidraw/issues/4884

the fix is 2 lines (`actionDistribute.tsx`). The rest are tests.

### CODE REVIEW

**Comment by zsviczian** [comment]
Looks good. 
Probably subject to another PR, but currently copy/paste style for a sticky note will only update the container and not the text within the container.

https://user-images.githubusercontent.com/14358394/157110264-a7096b90-1f71-4a8a-afee-026185813dc0.mp4

**Comment by dwelle** [comment]
> Probably subject to another PR, but currently copy/paste style for a sticky note will only update the container and not the text within the container.

Yes, let's fix that in another PR. I have one open from last year that improves style pasting in general.

**Review by zsviczian — APPROVED** [review]

File: src/actions/actionStyles.test.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+const { h } = window;
+
+const mouse = new Pointer("mouse");
+
+describe("actionStyles", () => {
+  beforeEach(async () => {
+    await render(<ExcalidrawApp />);
+  });
+  it("copy & pasting styles via keyboard", () => {
```
Comment (ad1992) [inline src/actions/actionStyles.test.tsx:17]:
```suggestion
  
  it("should copy & paste styles via keyboard", () => {
```

**Review by ad1992 — APPROVED** [review]

File: src/actions/actionStyles.test.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+const { h } = window;
+
+const mouse = new Pointer("mouse");
+
+describe("actionStyles", () => {
+  beforeEach(async () => {
+    await render(<ExcalidrawApp />);
+  });
+
```
Comment (dwelle) [inline src/actions/actionStyles.test.tsx:17]:
```suggestion
```

**Pull request merged**